documentation/JSON/schedule.json

```diff
@@ -0,0 +1,23 @@
+{
+  "disable": {
+    "start": "9:00",
+    "end": "17:00",
+    "days": [{
+      "sunday": false,
+      "monday": true,
+      "tuesday": true,
+      "wednesday": true,
+      "thursday": true,
+      "friday": true,
+      "saturday": false,
+      }]
```

Comment:
This is currently an array [] with a single dictionary {} inside. To make it easier to access elements, it should probably just be a dictionary.

Reply:
Good catch, I fixed it.